src/script/run_tox.sh

```diff
@@ -121,6 +121,10 @@ function main() {
     fi
 
     if [ ! -f ${venv_path}/bin/activate ]; then
+        if [ -d "$venv_path" ]; then
+            echo "$(readlink -f $venv_path) already exist, but is not a virtualenv. test_name empty?"
```

Comment:
syntax error. how is this connected to `test_name`?

Reply:
> how is this connected to test_name

If we're calling this script with `$prog_name --tox-envs py27,py3` and `tox_path` is empty and we're not giving this script a `test_name`, Line 118  will set `        venv_path="$build_dir/$test_name"` to `venv_path == $build_dir`. This is how I destroyed my build dir.

Reply:
> we're not giving this script a `test_name`

i see now.

> Where do you get a syntax error?

s/exist/exists/

also `readlink -f` is not portable. can we just put `$venv_path` ?  see https://www.freebsd.org/cgi/man.cgi?query=readlink .

Reply:
Fixed. Output now looks like so:

```
'/home/sebastian/Repos/ceph/src/script/../../build/' already exists, but is not a virtualenv. test_name empty?
```

Reply:
> '/home/sebastian/Repos/ceph/src/script/../../build/'

hmm, i admit that this is not ideal. probably an alternative is
```sh
cd $venv_path
echo "$PWD already exists, but is not a virtualenv. test_name empty?"
exit 1
```

Reply:
How about

```bash
function realpath() {
    # readlink -f is not portable
    python -c 'import os,sys;print(os.path.realpath(sys.argv[1]))' $1
+}
```
?